<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_SegmentationGUI</class>
 <widget class="QMainWindow" name="MainWindow_SegmentationGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Segmentation GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QVTKWidget" name="qVTKImg">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="checkBox_Image">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_Overlay">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Overlay Segmentation</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>582</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSet_Working_Directory"/>
    <addaction name="actionLoad_Image"/>
    <addaction name="actionSave_Image"/>
    <addaction name="actionClose_Image"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionVTK_Segment"/>
    <addaction name="actionITK_Segment"/>
    <addaction name="actionVXL_Segment"/>
    <addaction name="actionClear_all"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Image">
   <property name="text">
    <string>Load Image</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionVTK_Segment">
   <property name="text">
    <string>VTK Segment</string>
   </property>
  </action>
  <action name="actionITK_Segment">
   <property name="text">
    <string>ITK Segment</string>
   </property>
  </action>
  <action name="actionVXL_Segment">
   <property name="text">
    <string>VXL Segment</string>
   </property>
  </action>
  <action name="actionClear_all">
   <property name="text">
    <string>Clear all</string>
   </property>
  </action>
  <action name="actionSave_Image">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Segmented Image</string>
   </property>
  </action>
  <action name="actionSet_Working_Directory">
   <property name="text">
    <string>Set Working Directory</string>
   </property>
  </action>
  <action name="actionClear_Image">
   <property name="text">
    <string>Clear Image</string>
   </property>
  </action>
  <action name="actionClose_Image">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close Image</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QFrame</extends>
   <header>QVTKWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
